cmd/agent/app/server.go

```diff
@@ -83,6 +83,9 @@ func (a *Agent) runHealthServer(o *options.GrpcProxyAgentOptions) error {
 	muxHandler.Handle("/metrics", promhttp.Handler())
 	muxHandler.HandleFunc("/healthz", livenessHandler)
 	muxHandler.HandleFunc("/ready", readinessHandler)
+	// "/ready" is deprecated but being maintained for backward compatibility
+	muxHandler.HandleFunc("/ready", readinessHandler)
```

Comment:
Duplicate of line 84?

Reply:
Yeah, I think this is from rebase. Fixed now.